org.modeldriven.alf.eclipse.moka/META-INF/MANIFEST.MF

```diff
@@ -17,7 +17,7 @@ Require-Bundle: org.eclipse.uml2;bundle-version="4.0.2";resolution:=optional,
  org.eclipse.papyrus.moka;bundle-version="2.0.100",
  org.eclipse.papyrus.moka.fuml,
  org.eclipse.papyrus.moka.fuml.interfaces;bundle-version="2.0.0",
- org.eclipse.papyrus.moka.fuml.standardlibrary;bundle-version="2.0.0",
+ org.eclipse.papyrus.moka.fuml.standardlibrary;bundle-version="2.0.100",
```

Comment:
This was necessary as there was an API feature added after 2.0.0 that we rely on (Div_ class).